### ISSUE TITLE
1.19.4

### ISSUE THREAD

**Issue by Colonizor48** [kkllffaa/meteor-litematica-printer#26]
Make a ver for 1.19.4 please

**Comment by jjasim** [comment]
yeah there was a download link for 1.19.4 in old archives of the link, but now its no longer available. Please make the 1.19.4 version available for download again thanks
